elastic/kibana
# [Discover] Remove _type of  metaFields  to remove of sidebar

## Summary

We deprecated the `_type` field in 7.0.0: 
https://www.elastic.co/guide/en/elasticsearch/reference/7.17/removal-of-types.html

But it was still present in `metaFields` of uiSettings ( Configurable in Advanced Settings). 


<img width="974" alt="Bildschirmfoto 2022-06-17 um 13 45 29" src="https://user-images.githubusercontent.com/463851/174292186-4ec082bc-f7d7-40a8-b584-b79109924170.png">

Therefore it was available  and addable in Discover's Sidebar (You have to disable `Hide empty fields` for that):

<img width="328" alt="Bildschirmfoto 2022-06-17 um 13 41 41" src="https://user-images.githubusercontent.com/463851/174291681-f4af3e28-33d9-4d80-98cd-edf8431f218f.png">

Fixes #77209

### Testing
* Make sure in a new instance, there is no more `_type` field in `metaFields`.
* Make sure in the sidebar, when disabling `Hide empty fields`, there's no `_type` field.
* If user had a custom metaFields, then it should stay as it was set by the user, even if it had _type included

### Checklist
